github-upload.js

```diff
@@ -0,0 +1,27 @@
+import { Octokit, App } from "https://cdn.skypack.dev/octokit?dts";

```

Comment:
Would it not be better practice to import Octokit via node_modules?  I get that this works, but apparently importing from HTTP is still experimental.
https://nodejs.org/api/esm.html#https-and-http-imports

Reply:
Since my file was pretty small, I did it this way. I am not sure if there are any restrictions.